.dockerignore

```diff
@@ -1,6 +1,11 @@
 .git
 .vagrant
+**/_build
```

Comment:
Can you add these to the `.gitignore` too?

Reply:
good idea, will do !